monkey/infection_monkey/exploit/zerologon_utils/dump_secrets.py

```diff
@@ -0,0 +1,193 @@
+import io
+import logging
+import os
+import sys
+import traceback
+
+from impacket.examples.secretsdump import (LocalOperations, LSASecrets,
+                                           NTDSHashes, RemoteOperations,
+                                           SAMHashes)
+from impacket.smbconnection import SMBConnection
+
+LOG = logging.getLogger(__name__)
+
+
+# Adapted from https://github.com/SecureAuthCorp/impacket/blob/master/examples/secretsdump.py
```

Comment:
Read https://github.com/SecureAuthCorp/impacket/blob/master/LICENSE and do a proper copyright notice as they ask

Reply:
This applies to all examples

Reply:
todo

Reply:
Not fixed

Reply:
todo